pool/pool.py

```diff
@@ -77,7 +77,7 @@ def __init__(
         self.config = config
         self.constants = constants
 
-        if pool_config.get('store') == "MariadbPoolStore":
+        if pool_config["store"]["type"] == "mariadb":
```

Comment:
Can we use .get instead of brackets, it will return None if it's not available, which is the case for most people with the old config